go-controller/pkg/util/multi_network.go

```diff
@@ -1557,3 +1557,9 @@ func ParseNetworkName(networkName string) (udnNamespace, udnName string) {
 	}
 	return "", ""
 }
+
+// IsPrimaryNetworkCustomizationEnabled indicates if user defined IPs / MAC
+// addresses can be set in primary UDNs
+func IsPrimaryNetworkCustomizationEnabled() bool {
+	return IsNetworkSegmentationSupportEnabled() && config.OVNKubernetesFeature.EnableCustomNetworkConfig
+}
```

Comment:
Is it worth it to move this to the feature gate PR ?

Reply:
I don't care. Could go either way - but adding code that is not used feels weird.